### PR TITLE
Work about section

### PR DESCRIPTION
**Pull request by iamchristianani**
## Add the sections for Work  and About myself
I was able to do the following
- Updated the html file to add both the "Work" and "About myself" section
- Updated the style.css file while also making use of Grid and Flexbox
- Updated the img folder with more images
- Made use of Linters to check and fix errors

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -112,3 +112,283 @@ a {
...
+
+.work-section ul {
+  padding: 0;
+}
+
+.work-card li {
+  list-style: none;
+}
+
+.work-card-cont {
```
Comment (karlavdelgadof) [inline css/style.css:172]: Great job using `CSS Grid` to align your cards! :partying_face:

File: css/style.css [hunk 2]
```diff
@@ -112,3 +112,283 @@ a {
...
+  background-color: #008552;
+}
+
+.about-content button:hover {
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+}
+
+.about-content button:active {
+  background-color: #008552;
+}
```
Comment (karlavdelgadof) [inline css/style.css]: **[OPTIONAL]:** You could summarize these code and make it cleaner :sparkles:  by chaining selectors like so: 

```
work-card-info button:hover,
.about-content button:hover
 {
  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
}

.work-card-info button:active,
.about-content button:active {
  background-color: #008552;
}
```

**Note:** This is only a suggestion and It's the concept of a principle you will learn in further lessons called DRY. You can read a little about it [here :link: ](https://www.digitalocean.com/community/tutorials/what-is-dry-development?refcode=49d9dff6b635&utm_campaign=Referral_Invite&utm_medium=Referral_Program&utm_source=CopyPaste)

**Review by karlavdelgadof — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi there @iamchristianani  :smile: :technologist: !

You have successfully completed this project! There is nothing else to say other than... it's time to merge it! 🚀
Time to celebrate, you have done a fantastic job, congratulations! 🎉



## :sparkles: Highlights :sparkles:
- The section matches the provided design:rocket: :+1: 
- Great work using the right GitHub Flow :100: 
- Awesome file organization :card_file_box: 
- Commits are descriptive, good work. 
- Linters are working properly, great job!

APPROVED! 💯 💯 💯

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Cheers and best of luck with the following projects! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me **@karlavdelgadof**  in your question so I can receive the notification.
  
  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
  
  ------
  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (iamchristianani) [reply css/style.css:172]: Thank you very much, I appreciate your awesome review

File: css/style.css (hunk 2 above)
Comment (iamchristianani) [reply css/style.css]: Thank you for the suggestion. This has been updated accordingly.

**Pull request merged**